src/commands/scanner/run/dfa.ts

```diff
@@ -43,29 +42,30 @@ export default class Dfa extends ScannerRunCommand {
 			summary: getMessage(BundleName.RunDfa, 'flags.targetSummary'),
 			description: getMessage(BundleName.RunDfa, 'flags.targetDescription'),
 			delimiter: ',',
+			default: '.',
 			multiple: true
 		})(),
 		// END: Flags for targeting files.
 		// BEGIN: Config-overrideable engine flags.
 		'rule-thread-count': Flags.integer({
 			summary: getMessage(BundleName.RunDfa, 'flags.rulethreadcountSummary'),
-			description: getMessage(BundleName.RunDfa, 'flags.rulethreadcountDescription'),
+			default: 4,
```

Comment:
We can't set this to 4 by default either since the true definition of default (which we detect if null is detected) is:
    static final int DEFAULT_RULE_THREAD_COUNT =
            Math.min(Runtime.getRuntime().availableProcessors(), 4);
which ultimately depends on the user's machine's available processors.